web-animations-2/Overview.bs

```diff
@@ -2006,6 +2045,41 @@ partial interface AnimationTimeline {
 
 </div>
 
+<h3 id="the-animation-interface">The <code>Animation</code> interface</h3>
+
+Update the startTime and currentTime of the Animation interface as follows:
+
+<pre class="idl">
+[Exposed=Window]
+partial interface Animation : EventTarget {
+    attribute CSSNumberish?       startTime;
+    attribute CSSNumberish?       currentTime;
+};
+</pre>
+
+Replace [=start time=] and [=current time=] attribute descriptions with:
+
+<div class='attributes'>
+ :   <dfn attribute for=Animation>startTime</dfn>
+ ::  Update the description as:
+>    The [=start time=] of this animation. When associated with a
+>    [=progress-based timeline=], [=start time=] must be returned as a
+>    <a href="https://drafts.css-houdini.org/css-typed-om/#cssnumericvalue">
```

Comment:
Here and below, replace with {{CSSNumericValue}}.

Reply:
done